components/script/dom/domtokenlist.rs

```diff
@@ -131,8 +131,7 @@ impl DOMTokenListMethods for DOMTokenList {
 
     // https://dom.spec.whatwg.org/#stringification-behavior
```

Comment:
Fix to <https://dom.spec.whatwg.org/#dom-domtokenlist-stringifier> or <https://dom.spec.whatwg.org/#concept-dtl-serialize>, please. r+ with that.